packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/editForm.css

```diff
@@ -0,0 +1,17 @@
+.root {
+    display: grid;
+    grid-template-rows: 1fr min-content;
+    row-gap: 1.5rem;
+}
+
+.initialButtonContainer {
+    display: flex;
+    justify-content: center;
+}
+
+.updateButtonContainer {
+    display: grid;
+    grid-template-columns: min-content min-content;
```

Comment:
In general, `auto` is fine. There are uses for `min-content`, but they're rarer than you'd think. Consider reviewing the places you've used `min-content` and see if `auto` would be fine.

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/2280/commits/dd1654f50b2ffd5c467bf4b03f8a2f6631f5dfd5.